### DISCUSSION TITLE
Dark mode planned?

### DISCUSSION THREAD

**Discussion by Manoroc1** [pdfarranger/pdfarranger#1044] [General]
Love your work.
only thing i want you to add is dark mode.
is it planned or did i maybe missed the option?

**Comment by dreua** [comment]
Did you try entering "dark mode" in the github search or "dark mode Pdf arranger" in google?

It should work out of the box *and* there is even an option in the preferences, not sure what more we can do here for your eyes :wink: Make sure to use the latest version.

**Reply by dreua** [reply]
I mean do let us know if it doesn't work for you but then we need some more information about your os and versions to work with.
